social/email/61-email.js

```diff
@@ -55,7 +56,8 @@ module.exports = function(RED) {
         var smtpOptions = {
             host: node.outserver,
             port: node.outport,
-            secure: node.secure
+            secure: node.secure,
+            tls: {rejectUnauthorized: node.tls}
```

Comment:
just checking - letting this be null or undefined is OK ?

Reply:
Maybe we should check for null or at least set default value of true. Sorry, it is my very first touch to JavaScript.

Reply:
Yes - (hey no problem, glad you are learning). Yes - further up set it true by default - then test if n.tls === false - so only set it false if explicitly set to false by the user.